presto-hive/src/main/java/io/prestosql/plugin/hive/HdfsConfig.java

```diff
@@ -216,4 +217,17 @@ public HdfsConfig setFileSystemMaxCacheSize(int fileSystemMaxCacheSize)
         this.fileSystemMaxCacheSize = fileSystemMaxCacheSize;
         return this;
     }
+
+    public Integer getDfsReplication()
+    {
+        return dfsReplication;
```

Comment:
nit: I think it'd be good to still want to have the `@Min(1)` annotation, so that invalid values (eg. 0) can be detected during server startup. @agrawalreetika thoughts?

(The annotation seems to work for `Integer` class. It also lets null objects go through.)

Reply:
Thanks, @phd3 for your suggestion. This looks like a very valid to cover to avoid null.